src/inputs/date.html

```diff
@@ -5,6 +5,7 @@
 
 		<input
 			class="form-control"
+			datepicker-mode="picker.mode"
 			datepicker-options="{{picker.settings}}"
```

Comment:
is there a reason why we didnt just use the zn-datepicker directive available from zengine? cc @JoeBello @wizehive-chrismanjoine